htmd/queues/localqueue.py

```diff
@@ -115,8 +115,10 @@ def _createJobScript(self, fname, workdir, runsh, gpudevice=None):
             # Move completed trajectories
             if self.datadir is not None:
                 datadir = os.path.abspath(self.datadir)
-                if not os.path.isdir(datadir):
+                try:
                     os.mkdir(datadir)
+                except OSError:
```

Comment:
Do you try to catch the case that the directory already exists?

Reply:
yes

Reply:
There is a weird race condition when creating the directory on /tmp/ that Alberto hit upon.

Reply:
Well maybe OSError is too broad. I could catch FileExists error

Reply:
Or just use 
```python
os.makedirs(datadir, exist_ok=True)
```
without any exception catching...